### PR TITLE
MGMT-3752 controller shall print cluster operators status before exit…

### PR DESCRIPTION
**Pull request by tsorya**
…ing and sending last logs

### CODE REVIEW

File: src/assisted_installer_controller/assisted_installer_controller.go [hunk 1]
```diff
@@ -598,6 +598,19 @@ func (c controller) sendCompleteInstallation(isSuccess bool, errorInfo string) {
 	c.log.Infof("Done complete installation step")
 }
 
+// logClusterOperatorsStatus logging cluster operators status
+func (c controller) logClusterOperatorsStatus() {
+	operators, err := c.kc.ListClusterOperators()
```
Comment (romfreiman) [inline src/assisted_installer_controller/assisted_installer_controller.go:603]: Maybe describe as well?

Comment (tsorya) [reply src/assisted_installer_controller/assisted_installer_controller.go:603]: we decided to start with list, looks like we have all relevant info, if not will add more

Comment (eranco74) [reply src/assisted_installer_controller/assisted_installer_controller.go:603]: We get the status which is the relevant part of what you get in describe

**Review by eranco74 — COMMENTED** [review]
/lgtm

**Pull request merged**